sdk/identity/azure-identity/test/ut/token_credential_test.cpp

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include <gtest/gtest.h>
+
+#include <azure/core/test/test_base.hpp>
+#include <azure/identity/client_secret_credential.hpp>
+#include <azure/identity/environment_credential.hpp>
+
+#include <chrono>
+#include <thread>
+
+namespace Azure { namespace Identity { namespace Test {
+  class TokenCredentialTest : public Azure::Core::Test::TestBase {
+
+  protected:
+    // Required to rename the test propertly once the test is started.
```

Comment:
> propertly

Which did you mean here: property or properly?

Reply:
properly  or, the right way.

Reply:
```suggestion
// Required to rename the test properly once the test is started.
```